### DISCUSSION TITLE
Want to Contribute to Plane’s Globalization? Here's How! 🚀✨

### DISCUSSION THREAD

**Discussion by yasin172** [makeplane/plane#2510] [Ideas]
Hello 👋 Plane Developers,

First of all, I'd like to extend my deepest gratitude to everyone for your contributions to the Plane project and for the excellent technology choices. 👋  @pablohashescobar, @aaryan610, @anmolsinghbhatia, @dakshesh14, @NarayanBavisetti, @gurusainath, @sriramveeraghanta , @kunalv17, @vamsi and all those who have and will contribute... 

💡 I understand the significance of this project and am closely following its progress. I believe that adding multi-language support is crucial for reaching a broader audience. I think you should prioritize the development of  🚀 "Multi-Language" support.

✨If you decide to take steps in this direction, I can assist with the Turkish language translation. Considering the complex structure and linguistic nuances of Turkish, I think I can effectively contribute to accurate translation and integration. 

Thank you,
Yasin Atalay

**Comment by sriramveeraghanta** [comment]
Hello Yasin Atalay,

Thank you for your kind message and your keen interest in contributing to the Plane project. Your insight into the importance of multi-language support aligns with our vision for reaching a broader audience. Multi-language support is indeed on our roadmap, and we are actively working on the initial setup. Once we have completed the initial setup, we will reach out to you to begin the collaboration for Turkish translations.

Thank you once again for your support and enthusiasm.

**Reply by mutoulbj** [reply]
Hello，Sriram Veeraghanta，that's wonderful to hear!I think I can assist with the Chinese language translation and I'll do my best to assist.My team is in need of this feature.

**Comment by ludovicbarreca** [comment]
Hello,
I agree with OP, it is extremely important that you allow multi-language support. Right now, browsing the code shows there's no way to translate as all strings are as is, in English. This should not be overlooked and the technical debt of this method will keep increasing. I am ready to help on this matter, and could provide the help with my team as we are very interested in using Plane but this monolingual problem makes us unable to do so. Reach out to me if you want to collaborate on this - we could make this quickly happen (in a week or so).

**Comment by erdincdogrul** [comment]
Hello,
Likewise, I am one of those who think that multi-language support should be introduced. I am ready to support in this context

**Comment by rolife** [comment]
Dear colleagues,

I would like to draw your attention to the importance of multilingual support and specifically Russian language support in our work. As you know, Russian is one of the most widely spoken languages in the world and has a significant presence on the internet. By providing multilingual support, we can better serve our diverse audience and expand our reach.

I am happy to offer my assistance in this endeavor. I have experience in translating content from English to Russian and vice versa, as well as knowledge of localization best practices. Together, we can make our platform more accessible and inclusive for all users.

Thank you for your time and consideration. I look forward to working with you on this important project.

Sincerely,
Nikolai Roller.

**Comment by leandrogrupozani** [comment]
Olá.. Português do Brasil posso colaborar

**Reply by hpflatorre** [reply]
I've just sent a PR #6840 adding the Brazilian Portuguese language. Reviews are welcome @leandrogrupozani!
